Configuration/StandardSequences/python/AlCaRecoStreams_cff.py

```diff
@@ -175,6 +175,7 @@
 pathALCARECOSiStripCalMinBiasAAG = cms.Path(seqALCARECOSiStripCalMinBiasAAG*ALCARECOSiStripCalMinBiasAAGDQM)
 pathALCARECOSiStripCalSmallBiasScan = cms.Path(seqALCARECOSiStripCalSmallBiasScan)
 pathALCARECOSiStripCalZeroBias = cms.Path(seqALCARECOSiStripCalZeroBias*ALCARECOSiStripCalZeroBiasDQM)
+pathALCARECOSiStripCalZeroBias.replace(consecutiveHEs, unpackTcds*consecutiveHEs)
```

Comment:
why is this done here, instead of here: https://github.com/cms-sw/cmssw/blob/master/Calibration/TkAlCaRecoProducers/python/ALCARECOSiStripCalZeroBias_cff.py#L56 ?

Reply:
Fixed (will push in a while)

Reply:
I thought it was better to limit the substitution to DPGAnalysis code, instead of changing it for all the Calibration package. 

Can you please confirm it is fine to do it in Calibration/...?

Reply:
@vischia I am not sure to understand the argument about limiting the substitution to the DPGAnalysis code. Even here you are touching another package (Configuration/StandardSequences) and IMHO is better to change the subsystem packages (as Calibration) than the top-level configuration.

Reply:
@mmusich fair enough. I'll move the change to the relevant subsystem package.